Share The Load/Screens/ViewController.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  ViewController.swift
+//  Share The Load
+//
+//  Created by Christopher Hefferman on 10/01/2021.
+//
+
+import UIKit
+import Firebase
+import RealmSwift
+
+class ViewController: UIViewController {
+    
+    private let getRealmQuestionsButton = UIButton()
+    private let removeFromRealmButton = UIButton()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        fetchQuiz()
+        createGetRealmQuestionsButton()
+        createRemoveButton()
+    }
+    
+    
```

Comment:
To be really picky, can we just leave one empty line between funcs 😬

Reply:
Hit the easy one first, done. 👍🏼